### ISSUE TITLE
Dependency Updates - Main Batch - Kotlin/EventBus

### ISSUE THREAD

**Issue by ParaskP7** [wordpress-mobile/WordPress-Android#17560]
Parent #17551

This issue is about updating all `Main - Kotlin/EventBus` related dependencies for the whole project.

This `Main - Kotlin/EventBus` batch contains the following `2` dependencies:

- [x] `eventBusVersion = '3.3.1'` (see [releases](https://github.com/greenrobot/EventBus/releases)) -> [Comment](https://github.com/wordpress-mobile/WordPress-Android/issues/17560#issuecomment-1330683137)
- [x] `kotlinxCoroutinesVersion = '1.5.2'` (see [releases](https://github.com/Kotlin/kotlinx.coroutines/releases)) #17673

**Comment by ParaskP7** [comment]
✅ The latest stable version of `eventBusVersion` is `3.3.1` (see [releases](https://github.com/greenrobot/EventBus/releases)). As such, this item is now marked as complete.

**Comment by ParaskP7** [comment]
👋 @oguzkocer !

FYI: I took the liberty to unassign you from this batch update issue, and assign myself instead. I did that in order to start progress on the now blocking `kotlinxCoroutinesVersion` update to `1.6.4` (for some more context see [draft PR](https://github.com/wordpress-mobile/WordPress-Android/pull/17643)). In a nutshell, the `kotlinxCoroutinesVersion` update is now blocking the whole #17558 batch update.
